### DISCUSSION TITLE
There is $watch... is there a way to 'un-watch' ?

### DISCUSSION THREAD

**Discussion by anentropic** [alpinejs/alpine#4314] [1. Help]
Basically what's in the title.

We can:
```html
<div x-data="{ open: false }" x-init="$watch('open', value => console.log(value))">
    <button @click="open = ! open">Toggle Open</button>
</div>
```

to 'watch' a data property and execute some code when it changes

is there a way to 'unwatch'?

**Comment by ekwoka** [comment]
Yes, just remove the property.

**Reply by anentropic** [reply]
which property on what?

**Reply by ekwoka** [reply]
well, really the element.

the effect is bound to the element.
